ConnectivityDemos/ConnectivityDemosTest/pom.xml

```diff
@@ -67,6 +67,13 @@
     		<scope>system</scope>
     		<systemPath>${project.basedir}/lib_ws/client/cxfClient_1624D62D6426F325.jar</systemPath>
 		</dependency>
+		
+		<dependency>
+    		<groupId>org.assertj</groupId>
```

Comment:
whitespaces mix (tabs, spaces). Breaks formating

Reply:
no fixed; but applied to the full pom.xml - so I do not have to apply surgery changes ... especially since this pom.xml already contained mix intentions.